packages/gatsby-transformer-remark/utils/get-heading-id.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+exports.__esModule = true;
+exports.getHeadingID = void 0;
+
+const getHeadingID = heading => {
+  const data = heading.data;
+
+  if (data) {
+    if (data.id) return data.id;
+
+    if (data.htmlAttributes && data.htmlAttributes.id) {
+      return data.htmlAttributes.id;
+    }
+
+    if (data.hProperties && data.hProperties.id) {
+      return data.hProperties.id;
+    }
+  }
+
+  return null;
+};
+
+exports.getHeadingID = getHeadingID;
```

Comment:
Hey, that's not `src` file - it's transpiled one :) - removing first few lines from it and moving to `src/utils/` should handle that

Reply:
My bad. I didn't notice it.